dev-requirements.txt

```diff
@@ -2,5 +2,6 @@ ipython==3.1.0
 ipdb==0.8.1
 pytest==2.7.1
 pytest-cov==1.8.1
+Mock=1.0.1
```

Comment:
I think this needs to be lower-case, and needs a double equals. For future reference, an easy way to get the correct formatting for requirements files is 
```pip freeze```
you can find the specific requirement with
```
pip freeze | grep <package-name>
```
ex:
```
pip freeze | grep mock
```
and you can append it to the file with
```
pip freeze | grep <package-name> >> dev-requirements.txt
```